packages/carousel/src/react/index.js

```diff
@@ -144,17 +149,30 @@ Pages.propTypes = {
 }
 
 function Page({ isActivePage, ...props }) {
+  const ref = React.useRef()
   const { offset } = React.useContext(CarouselContext)
 
   return (
     <li
+      ref={ref}
       {...styles.page()}
       {...css({ transform: `translate3d(${offset}px, 0, 0)` })}
-      {...(!isActivePage && { 'aria-hidden': true, tabIndex: -1 })}
       {...props}
-    />
+      {...(!isActivePage && { hidden: true, tabIndex: -1 })}
+    >
+      <Transition in={isActivePage} timeout={500}>
```

Comment:
let's see, does this do as follows:

- When page is active, transitions "in"
- When "in", runs child render prop
- If was "in" but now leaving, "exited", then is gone again
- Else, it's "in", and active page children are shown

Reply:
And, in the commit message, this dom removal was for "better keyboard nav experience".  What exactly is better about it?

Reply:
only the active page is accessible by keyboard. i was trying to do a bunch of different things to make this happen but i've concluded that the best approach is only render what's active.